docs/dyn/adexchangebuyer2_v2beta1.accounts.clients.invitations.html

```diff
@@ -105,11 +105,11 @@
     "clientAccountId": "A String", # Numerical account ID of the client buyer
         # that the invited user is associated with.
         # The value of this field is ignored in create operations.
+    "invitationId": "A String", # The unique numerical ID of the invitation that is sent to the user.
```

Comment:
@saicheems Sorry I have not had a chance to investigate this further; filed https://github.com/google/google-api-python-client/issues/386.